komand/server.py

```diff
@@ -98,13 +120,53 @@ def handler(prefix, name, test):
                 r.status_code = status_code
                 return r
 
+        @app.route("/api_spec")
+        def api_spec():
+            """API spec details endpoint.
+            ---
+            get:
+              description: Get API spec details
+              responses:
+                200:
+                  content:
+                    application/json:
+                      type: object
+            """
+
+            return self.spec.to_yaml()
+
+        @app.route("/info")
+        def plugin_spec():
```

Comment:
Rename to `plugin_info` as we aren't returning the schema, just info about the plugin.
```suggestion
        def plugin_info():
```